core/tests/test_utils.py

```diff
@@ -3697,13 +3731,15 @@ def save_new_topic(
         Returns:
             Topic. A newly-created topic.
         """
+        canonical_story_ids = canonical_story_ids or []
+        additional_story_ids = additional_story_ids or []
         canonical_story_references = [
             topic_domain.StoryReference.create_default_story_reference(story_id)
-            for story_id in (canonical_story_ids or [])
+            for story_id in canonical_story_ids
         ]
         additional_story_references = [
             topic_domain.StoryReference.create_default_story_reference(story_id)
-            for story_id in (additional_story_ids or [])
+            for story_id in additional_story_ids
```

Comment:
Any specific reason for this changes?

Reply:
@DubeySandeep I moved the assigning of the default value `[]` to `canonical_story_ids` and `additional_story_ids` into its own line above in case any new features are added that depends on either list. It was convenient for the new field I added at the time, but now that field was moved to a different storage model.